extensions/community_extensions/skadnetwork.md

```diff
@@ -439,6 +439,8 @@ If the bid request included the `BidRequest.imp.ext.skadn` object, then a DSP co
       <td>
         SKAdNetwork signature as specified by Apple
         </br></br>
+        <strong>Note</strong>: Apple requires that both the ad network nonce and ad network identifier be lowercase when signing for either fidelity type (impressions or clicks), as per SKAdNetwork specifications.
+        </br></br>
```

Comment:
In our testing (at AdColony), we found that we only got SKAN postbacks when the ad network identifier used to generate the SKAdNetwork signature (and passed to [`loadProduct(withParameters:completionBlock:)`](https://developer.apple.com/documentation/storekit/skstoreproductviewcontroller/1620632-loadproduct) was a case match with whatever the source app put in their `Info.plist`.

We send SKAN network IDs out as they are in an app's `Info.plist` so that buyers can generate correct signatures, but someone who read this might think they need to lower-case their network ID always — even when it would not match the `Info.plist` value.